css/main.css

```diff
@@ -5,6 +5,25 @@ a { text-decoration:none;}
 .navbar, .jumbotron {
     margin-bottom:0;
     border-radius: 0;
+}
+.navbar {
+    background-color:crimson;
```

Comment:
You should try to not modify the classes from bootstrap. If you want a special style, you should create a new class.